internal/services/paloalto/next_generation_firewall_vhub_local_rulestack_resource.go

```diff
@@ -29,10 +29,12 @@ type NextGenerationFirewallVHubLocalRuleStackModel struct {
 	ResourceGroupName string `tfschema:"resource_group_name"`
```

Comment:
Suggest triggering the [related test cases](https://github.com/hashicorp/terraform-provider-azurerm/tree/main/internal/services/paloalto) for the changed resources on your local environment to ensure there is no regression issue. Please also paste the test result in this PR. Thanks.